src/index.js

```diff
@@ -303,18 +302,23 @@ class Dayjs {
     })
   }
 
+  utcOffset() {
+    return -Math.round(this.$d.getTimezoneOffset() / 15) * 15
```

Comment:
why `15` here, can this be any other number?

Reply:
In the `momentjs` code, there was this comment

> // On Firefox.24 Date#getTimezoneOffset returns a floating point.
    // https://github.com/moment/moment/pull/1871

It seems there was a bug with FF24, and in order to avoid issues with the floating point, they round it around 15, which are minutes.

I'm not fully sure about how this solves a possible bug, but `momentjs` is fully tested, and I think we can trust its code. Maybe we should add some sort of comment?

Reply:
I see, thanks. And add some comments please.